bodhi/server/models.py

```diff
@@ -759,6 +760,9 @@ class Release(Base):
             associated with this release.
         composed_by_bodhi (bool): The flag that indicates whether the release is composed by
             Bodhi or not. Defaults to True.
+        create_automatic_updates (bool): A flag indicating that updated should
```

Comment:
We need a way to set this boolean using the cli for example. So that when Release engineering creates a new release they can set this boolean.

Reply:
I'm working on this one, this needs additions/updates in a couple of places (e.g. tests).